Swift/BRCryptoTests/BRCryptoCommonTests.swift

```diff
@@ -98,16 +99,44 @@ class BRCryptoCommonTests: XCTestCase {
         //
         // Phrase
         //
-        k = Key.createFrom (phrase: "nothing", words: BRCryptoAccountTests.words)
-        XCTAssertNil(k)
         k = Key.createFrom (phrase: "ginger settle marine tissue robot crane night number ramp coast roast critic", words: BRCryptoAccountTests.words)
         XCTAssertNotNil(k)
+        k = Key.createFrom (phrase: "ginger settle marine tissue robot crane night number ramp coast roast critic", words: nil)
+        XCTAssertNil(k)
+        k = Key.createFrom (phrase: "not-a-chance", words: BRCryptoAccountTests.words)
+        XCTAssertNil(k)
 
         // Pigeon
+        s = "5Kb8kLf9zgWQnogidDA76MzPL6TsZZY36hWXMssSzNydYXYB9KF"
+        k = Key.createFromString(asPrivate: s)
+        n = "nonce".data(using: .utf8)
+        l = Key.createForPigeonFrom(key: k, nonce: n)
```

Comment:
Do we want to assert on this being not nil?

Reply:
Added a check.